### PR TITLE
OSDOCS-2667: Add an architecture diagram

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://issues.redhat.com/browse/OSDOCS-2667

Link to docs preview: https://52289--docspreview.netlify.app/openshift-enterprise/latest/architecture/architecture.html#architecture-platform-introduction_architecture

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: updating/updating-cluster-rhel-compute.adoc [hunk 1]
```diff
@@ -18,6 +18,7 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
...
 * If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being updated to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[_Upgrading an OpenShift Container Platform cluster configured for manual mode with STS_].
+* If you run an Operator or you have configured any application with the Pod Disruption Budget, you might experience a block during the upgrade process. The `PodDisruptionBudget` with `minAvailable: 1`  may block the eviction process while nodes are drained to apply pending MachineConfigs. If several nodes are rebooted, all the pods may run on only one node, and the `PodDisruptionBudget` can prevent the node drain.
```
Comment (zanetworker) [inline updating/updating-cluster-rhel-compute.adoc:21]: How is this related to the architecture diagram?

**Review by zanetworker — COMMENTED** [review]
Diagram LGTM

File: updating/updating-cluster-rhel-compute.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/updating-cluster-rhel-compute.adoc]: @zanetworker I fixed it. sorry, the irrelevant commit seemed to be added in this PR.

**Comment by zanetworker** [comment]
LGTM

**Review by skrthomas — COMMENTED** [review]
/lgtm
